test/aws.tf

```diff
@@ -0,0 +1,23 @@
+# Default
+provider "aws" {
+  version = "2.16.0"
+  region  = "us-east-1"
+}
+
+provider "aws" {
+  version = "2.16.0"
```

Comment:
btw we don't need to set version for each provider, because multiple versions of the same provider is not supported. It is enough to set it to the first one. Also there are newer versions available, i guess we can bump it to the last one

Reply:
let's keep it explicit for now